### PR TITLE
[HUST CSE] fix: Fixed wrong download link of env in 20+ files, and so…

### PR DESCRIPTION
**Pull request by Hornos3**
…me other wrong links.

## 拉取/合并请求描述：(PR description)

[

#### 为什么提交这份PR (why to submit this PR)
原仓库中超过20个文件的env下载链接不正确，页面会引导到主页面。

#### 你的解决方案是什么 (what is your solution)
修改错误的env下载链接。（https://www.rt-thread.org/download.html#download-rt-thread-env-tool）

#### 在什么测试环境下测试通过 (what is the test environment)

]

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [ ] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [ ] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [ ] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [ ] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [ ] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [ ] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [ ] 代码是高质量的 Code in this PR is of high quality
- [ ] 已经使用[formatting](https://github.com/mysterywolf/formatting)等源码格式化工具确保格式符合[RT-Thread代码规范](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_cn.md) This PR complies with [RT-Thread code specification](https://github.com/RT-Thread/rt-thread/blob/master/documentation/contribution_guide/coding_style_en.md)

### CODE REVIEW

File: bsp/allwinner_tina/README.md [hunk 1]
```diff
@@ -126,6 +126,6 @@ msh />
 [uestczyh222][4] < [lymz@foxmail.com][5] >
 
 
-  [1]: https://www.rt-thread.org/page/download.html
+  [1]: https://www.rt-thread.org/download.html
```
Comment (shandianchengzi) [inline bsp/allwinner_tina/README.md]: 可以加锚点。https://www.rt-thread.org/download.html#download-rt-thread-env-tool

File: bsp/acm32/acm32f4xx-nucleo/README.md [hunk 2]
```diff
@@ -51,6 +51,6 @@ ACM32F4xx芯片是上海爱信诺航芯电子科技有限公司(后续简称上
 * 板子[数据手册][2]
 * 芯片[数据手册][3]
 
-  [1]: https://www.rt-thread.org/page/download.html
+  [1]: https://www.rt-thread.org/download.html
```
Comment (shandianchengzi) [inline bsp/acm32/acm32f4xx-nucleo/README.md]: https://www.rt-thread.org/download.html#download-rt-thread-env-tool

File: bsp/acm32/acm32f0x0-nucleo/README.md [hunk 3]
```diff
@@ -53,6 +53,6 @@ ACM32F0x0芯片是上海爱信诺航芯电子科技有限公司(后续简称上
 * 板子[数据手册][2]
 * 芯片[数据手册][3]
 
-  [1]: https://www.rt-thread.org/page/download.html
+  [1]: https://www.rt-thread.org/download.html
```
Comment (shandianchengzi) [inline bsp/acm32/acm32f0x0-nucleo/README.md]: https://www.rt-thread.org/download.html#download-rt-thread-env-tool

File: bsp/bouffalo_lab/README.md [hunk 4]
```diff
@@ -179,7 +179,7 @@ Windows下推荐使用[env工具][1]，在console下进入bsp/bouffalo_lab/bl61x
 
 * 芯片[datasheet][2]
 
-  [1]: https://www.rt-thread.org/page/download.html
+  [1]: https://www.rt-thread.org/download.html
```
Comment (shandianchengzi) [inline bsp/bouffalo_lab/README.md]: https://www.rt-thread.org/download.html#download-rt-thread-env-tool

**Review by shandianchengzi — APPROVED** [review]

**Comment by shandianchengzi** [comment]
https://github.com/RT-Thread/rt-thread/issues/7270

**Review by shandianchengzi — APPROVED** [review]

**Review by mysterywolf — APPROVED** [review]

**Pull request merged**